spring-cloud-gcp-data-spanner/src/main/java/com/google/cloud/spring/data/spanner/core/admin/SpannerSchemaUtils.java

```diff
@@ -186,8 +186,13 @@ String getColumnDdlString(SpannerPersistentProperty spannerPersistentProperty,
 					spannerPersistentProperty.isGenerateSchemaNotNull(),
 					spannerPersistentProperty.isCommitTimestamp());
 		}
-		spannerJavaType = spannerEntityProcessor
+		if (spannerColumnType != null && spannerColumnType.equals(Type.Code.JSON)) {
```

Comment:
Can this logic go into `spannerEntityProcessor.getCorrespondingSpannerJavaType`?

Reply:
I see that moving logic inside would be more readable. but I don't see a direct way of doing it, since `spannerEntityProcessor` have no access to either `spannerPersistentProperty` or `spannerPersistentEntity`, thus has no info about the annotation. And `getCorrespondingSpannerJavaType` is a public method, I don't know if I want to change and add parameter to it?